### DISCUSSION TITLE
hpool.com has over 51% of the network storage space

### DISCUSSION THREAD

**Discussion by joelstiller** [Chia-Network/chia-blockchain#4892] [General]
These plots need to be straight up banned off the network. Pool farming should NOT be allowed until it's available for everyone, and natively supported.

**Comment by gyarros** [comment]
exactly

**Comment by dyorgio** [comment]
To me, it is a [ponzy scheme](https://en.wikipedia.org/wiki/Ponzi_scheme)... a home farmer (1 - 2TB of plots) can only get XCH after 253 days currently... (minimum of 0.2 XCH to transfer)

**Comment by joelstiller** [comment]
I mean, not it's not a ponzy scheme. At current coin price, even with transfer fee. That's still 7.11$ a day for a 2TB hard drive. Well in line with what an eth miner with a 3080 makes, with far less cost overhead via power. The issue however, is in something that should be 'decentralized' no one entity should have that much control over the network.

**Comment by ilunaryStar** [comment]
> To me, it is a [ponzy scheme](https://en.wikipedia.org/wiki/Ponzi_scheme)... a home farmer (1 - 2TB of plots) can only get XCH after 253 days currently... (minimum of 0.2 XCH to transfer)

You re not small farmer.You gotta have at least 20 tb.

**Comment by MTXDevp** [comment]
hpool.com 51% attack incoming, rip chia

**Comment by joelstiller** [comment]
Like, how is this ok. 

Blocks won within last 10k 3,585 (35.85%) (1,567.2 PiB) |  Blocks won in last 24H 1,624 | xch1f0ryxk6qn096hefcwrdwpuph2hm24w69jnzezhkfswk0z2jar7aq5zzpfj

**Comment by dyorgio** [comment]
> I mean, not it's not a ponzy scheme. At current coin price, even with transfer fee. That's still 7.11$ a day for a 2TB hard drive. Well in line with what an eth miner with a 3080 makes, with far less cost overhead via power. The issue however, is in something that should be 'decentralized' no one entity should have that much control over the network.

I mean... 243 days is a lot of time for wait, much people enter (small), nobody transfer yet.

**Comment by michael-trinity** [comment]
This should be taken care of quickly, hope they ban the whole pool

**Comment by baizuchong-net** [comment]
这应当得到重视

**Comment by pluto2021** [comment]
Exactly! No pool shall ever be able to have 51% of netspace share. When this % is approaching, the pool size shall be frozen until its % share would drop below a threshold.  Sooner or later, all solo would want to join the largest pool to get the most bang of the buck, like it or not and the entire net space could soon be dominated by large monster pools that can take say 80-90% of the power. 

Remember that Aisa has most of the HDD manufacturing capability, they will surely dominate the pool and if there is no cap limit, they would take over majority of the rewards in no time, I mean, majority not just 51%, if no protocol to limit pool size.  The decentralized network will soon be centralized in Asia, defeating the scheme.  Not sure if this geological imbalance can still be taken into account when updating the blockchain software.  Essentially we want Chia to be geologically balanced somehow.

**Reply by joelstiller** [reply]
Why not just block pooling entirely? There is no reason to allow such centralization.
